definitions/procedures/pulpcore/container_handle_image_metadata.rb

```diff
@@ -0,0 +1,23 @@
+module Procedures::Pulpcore
+  class ContainerHandleImageMetadata < ForemanMaintain::Procedure
+    include ForemanMaintain::Concerns::SystemService
+    include ForemanMaintain::Concerns::PulpCommon
+
+    metadata do
+      description 'Initialize and expose container image metadata in the pulpcore db'
+      for_feature :pulpcore
+    end
+
+    def run
+      with_spinner('Initialize and expose container image metadata in the pulpcore db') do |spinner|
+        necessary_services = feature(:pulpcore_database).services
+
+        feature(:service).handle_services(spinner, 'start', :only => necessary_services)
+
+        spinner.update('Adding image metadata to pulp. You can continue using the ' \
+               'system normally while the task runs in the background.')
+        execute!(pulpcore_manager('container-handle-image-data'))
+      end
```

Comment:
Wondering if we can add a step to run the [rake task](https://github.com/Katello/katello/pull/10988/files#diff-055c9a3444ca692a4bf7d89c9e46f3495096c13c3b0b2355a731086e1128d116R75) to reindex katello data right after this in the file like 
`execute!('foreman-rake katello:import_docker_manifest_labels')`

Reply:
I think indexing the content right after running this would be ideal since it has to happen anyway. It's just a matter of us having one or two commands in the documentation.

Reply:
I'll update it here..

Reply:
Ok..I will let this PR move independently of the other PR for now and make any updates to this file when the rake task is merged.